src/util/account.js

```diff
@@ -52,6 +46,13 @@ export function validateSeed (seed) {
     }
   }
 
+  if (!validBip39Seed) {
+    return {
+      valid: false,
+      reason: `This recovery phrase will be treated as a "legacy Parity brain wallet"`
```

Comment:
Some wording suggestions (but please take with grain of salt):

 - Parity brain wallets are no longer supported.
 - The recovery phrase you entered is not a valid BIP39 phrase.
 - The recovery phrase you entered is a legacy Parity brain wallet seed, and is no longer supported.